NuGet.sln

```diff
@@ -222,12 +222,14 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NuGet.Shared.Tests", "test\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NuGet.MSSigning.Extensions.FuncTest", "test\NuGet.Clients.FuncTests\NuGet.MSSigning.Extensions.FuncTest\NuGet.MSSigning.Extensions.FuncTest.csproj", "{BA7F681E-FDC7-42B6-8BF1-74DA6421BA1A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NuGet.MSSigning.Extensions.Test", "test\NuGet.Clients.Tests\NuGet.MSSigning.Extensions.Test\NuGet.MSSigning.Extensions.Test.csproj", "{2A604A43-C72A-4A03-B68B-66AF4B6F430E}"
```

Comment:
Why is the project Guid changing here?

Reply:
Also updating the guid for test projects based off of this [comment](https://github.com/dotnet/project-system/issues/3079#issuecomment-354180353).